docs: remove constellation-state.json (#469)
edgelesssys/constellation: docs/docs/architecture/orchestration.md

The `kubeconfig` file provides the credentials and configuration for connecting and authenticating to the API server.
Once configured, orchestrate the Kubernetes cluster via `kubectl`.

> Keep the state files in the workspace directory such as the `terraform.tfstate` for the CLI to be able to manage your cluster.

Reviewer: ```suggestion
Make sure to keep the state files such as `terraform.tfstate` in the workspace directory to be able to manage your cluster later on.
```

Original sentence already sounds kinda awkward to me; here's a suggestion for a better one?
